jsonschema/apis/Files_v1_000.json

```diff
@@ -0,0 +1,214 @@
+{
+  "openapi": "3.0.1",
+  "servers": [
+    {
+      "description": "API para a entidade Products (Produtos) do segmento HealthCare para produtos TOTVS",
+      "url":"http://{serverUrl}:{serverHttpPort}/api/healthcare/central/v1",
+      "variables":{
+        "serverUrl":{
+          "default":"localhost"
+        },
+        "serverHttpPort":{
+          "default":"4202"
+        }
+      }
+    }
+  ],
+  "info": {
+    "description": "API para a entidade files (arquivos SBX) do segmento HealthCare para produtos TOTVS",
+    "version": "1.000",
+    "title": "Files",
+    "contact": {
+      "name": "T-Talk",
+      "url": "http://api.totvs.com.br",
+      "email": "comiteintegracao@totvs.com.br"
+    },
+    "x-totvs": {
+      "messageDocumentation": {
+        "name": "Health Files",
+        "description": "Files SBX",
+        "segment": "HealthCare"
+      },
+      "productInformation": [
+        {
+          "product": "central-obrigacoes",
+          "contact": "",
+          "description": "Histórico de Arquivos SBX",
+          "adapter": "plsmvcarqsib.prw"
+        }
+      ]
+    }
+  },
+  "paths": {
+    "/files/{yearMonthRefer}": {
+      "get": {
+        "tags": [
+          "collection"
+        ],        
+        "parameters": [
+          {
+            "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Authorization"
+          },
+          {
+            "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Order"
+          },
+          {
+            "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Page"
+          },
+          {
+            "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/PageSize"
+          },
+          {
+            "$ref":"https://raw.githubusercontent.com/totvs/ttalk-standard-message/master/jsonschema/apis/types/totvsApiTypesBase.json#/parameters/Fields"
+          },
+          {
+            "$ref": "#/components/parameters/yearMonthRefer"
+          },
+          {
+            "$ref":"#/components/parameters/healthInsurerCode"
+          }
+        ],
+        "summary": "Busca os arquivos de uma competência",
+        "description": "Retorna os arquivos de uma competência",
+        "operationId": "getFiles",
+        "responses": {
```

Comment:
Sugiro incluir o código 401 (unauthorized) para sinalizar erros de autenticação (usuário ou senha errados) quando o cabeçalho Authorization for utilizado.

Reply:
Incluído